README.md

```diff
@@ -113,22 +113,25 @@ wrapperUpgrade {
             dir = ...
             options {
                 gitCommitExtraArgs = [...]
-                allowPreRelease = [...]
+                allowPreRelease = true
+                labels = ['dependencies']
+                ignoreClosedPullRequests = true
```

Comment:
IMO the name `ignoreClosedPullRequests` is not quite right. I feel a better name would be `recreateClosedPullRequests` which is much less ambiguous to me.